README.md

```diff
@@ -97,8 +97,9 @@ You can load many more quantized models hosted on the hub under the Intel organi
 Check out the [`examples`](https://github.com/huggingface/optimum-intel/tree/main/examples) directory for more sophisticated usage.
 
 ### OpenVINO
+Below are the examples of how to use OpenVINO and its [NNCF](https://docs.openvino.ai/latest/tmo_introduction.html) framework for model optimization, quantization, and inference.
 
-Here is an example on how to perform inference with OpenVINO Runtime:
+- OpenVINO inference example:
```

Comment:
```suggestion
#### OpenVINO inference example:
```
IMO it looks nicer if this is a header, and it makes it possible to create a direct link to an example.